### PR TITLE
Run2-alca117 Modify all macros for IsoTrack Calibration

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibFitPlots.C [hunk 1]
```diff
@@ -427,63 +436,78 @@ void FitHistExtended(const char* infile, const char* outfile,std::string prefix,
...
-	    fitOneGauss(hist2,debug);
-	    hists.push_back(hist2);
-	    std::pair<double,double> meaner0 = fitTwoGauss(hist,debug);
-	    value = meaner0.first;
-	    error = meaner0.second;
-	  } else {
-	    std::pair<double,double> meaner = fitOneGauss(hist,debug);
-	    value = meaner.first; error = meaner.second;
+	double value(0), error(0), total(0);
+	if (hist1 == 0) {
```
Comment (ghellwig) [inline Calibration/HcalCalibAlgos/macros/CalibFitPlots.C]: please use `nullptr` (and also below where appropriate)

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ghellwig** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**